mkt/comm/models.py

```diff
@@ -134,21 +134,22 @@ def user_has_perm_note(note, profile, request=None):
     Developers of the add-on used in the note, users in the CC list,
     and users who post to the thread are allowed to access the object.
 
-    Moreover, other object permissions are also checked agaisnt the ACLs
+    Moreover, other object permissions are also checked against the ACLs
     of the user.
     """
     if note.author and note.author.id == profile.id:
         # Let the person access their own note.
         return True
 
-    if note.note_type == comm.DEVELOPER_COMMENT:
-        # Developer comment only for developers.
-        return note.thread.check_obj_author(profile)
```

Comment:
it's handled down here https://github.com/mozilla/zamboni/pull/3432/files#diff-48fdb15f6d3232f4ed1f0259b11f1952R154 ... i'm not sure what i was thinking when i added these lines, probably thinking about write access instead of read. i'll add more tests.